source/guide_mautic.rst

```diff
@@ -0,0 +1,381 @@
+
+.. note::
+  This is necessary, because your uberspace comes with a number of :manual:`security headers <web-security-headers>` set by default.
+  In this particular case ``X-Frame-Options: SAMEORIGIN`` is set.
+  It causes problems, because it prevents Mautic's forms that are embedded on your main site to receive form result data.
+  This leads to an error that the `form keeps hanging on "please wait" <https://forum.mautic.org/t/multisite-wordpress-trouble-mautic-form-iframe-error-refused-to-connect-please-wait-hanging-mautic-form-submission/11092>`_ after it was submitted.
+
+Suppressing `X-Frame-Options: SAMEORIGIN`
+-----------------------------------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace web header suppress mautic.mysite.com X-Frame-Options
+  Suppressing header "X-Frame-Options" for mautic.mysite.com
+  [isabell@stardust ~]$
+
+
+Finishing installation
+======================
+
+Now you can access the Mautic's web interface and the installation wizard to complete your setup.
+You retrieved your MySQL username and password in the `Prerequisites` section and set the database name there (e.g. ``isabell_mautic``).
+Use these values for the wizard.
+
+Set up Mautic's cronjobs
+-----------------------
+
+Mautic requires a few `cron jobs <https://docs.mautic.org/en/setup/cron-jobs>`_ to handle some maintenance tasks such as updating contacts or campaigns, executing campaign actions, sending emails, and more.
+You must manually add the required :manual:`cron jobs to your server<daemons-cron>`.
+
+.. important::
+  This guide assumes the following:
+
+  1. You're Mautic instance is installed in: ``/var/www/virtual/$USER/mautic``.
+  2. You use the `email queue <https://docs.mautic.org/en/contacts/message-queue>`_ (`Queue documentation <https://docs.mautic.org/en/queue>`_). If you don't you can omit the line containing ``mautic:emails:send``.
+  3. You get your `bounces <https://docs.mautic.org/en/channels/emails/bounce-management>`_ from somewhere. If you don't or handle them somehow by yourself you can omit the line containing ``mautic:emails:fetch``.
+  4. You are subject to the European GDPR regulation. If you sure you aren't (*which is highly unlikely given it's architecture*) you could omit the the line containing ``mautic:maintenance:cleanup --days-old=365``. This simply removes data older than one year, which is probably advisable regardless of the GDPR.
+
+We save the logs of Mautic's console. If you don't want this remove the part after ``>>`` in your crontab.
+If you keep the logs create the directory where the logs will be saved to:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mkdir -p /home/$USER/logs/mautic
+  [isabell@stardust isabell]$
+
+
+To create your cron instructions. You can use the following command:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cat << EOF
```

Comment:
Dont do it like this. Just provide a codeblock (or multiple codeblocks, like described in my commend about the steps in line 280-282) with the cronjobs and tell the readers to add this to their cronjob. Add a link to the manual. See for example how its done here: https://lab.uberspace.de/guide_restic/#cronjob

Reply:
Done, but I don't feel that his enhances the quality of the guide. Instead of copying just one output, opening crontab, pasting the whole output and _maybe_ deleting some of the lines, people need to switch between editing crontab and the bash instance back and forth.